modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common registry terms
+
+This glossary defines common terms that are used in the registry content. 
+
+container::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
+
+Image Registry Operator::
+The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
+
+image repository::
+An image repository is a collection of related container images and tags identifying images.
+
+mirror registry::
+A registry that holds the mirror of {product-title} images.
```

Comment:
You can start with `The mirror registry is a registry....` to stay consistent with the other definitions or you can modify other terms to be consistent with this one.

Reply:
acked